### PR TITLE
Fix ClipDepth and Name parsing order

### PR DESCRIPTION
**Pull request by r0mai**
The swf spec says Name comes first then ClipDepth.

### CODE REVIEW

**Comment by r0mai** [comment]
sorry, wrong target repository :)

File: lib/modules/swfobject.c [hunk 1]
```diff
@@ -79,9 +79,8 @@ static int objectplace(TAG * t,U16 id,U16 depth,MATRIX * m,CXFORM * cx,const cha
   if (flags&PF_MATRIX) swf_SetMatrix(t,m);
   if (flags&PF_CXFORM) swf_SetCXForm(t,cx,1);
   if (flags&PF_RATIO) swf_SetU16(t,0);
-  /* ??? The spec states that name comes first? */
-  if (flags&PF_CLIPDEPTH) swf_SetU16(t, clipaction);
   if (flags&PF_NAME) swf_SetString(t,name);
+  if (flags&PF_CLIPDEPTH) swf_SetU16(t, clipaction);
```
Comment (matthiaskramm) [inline lib/modules/swfobject.c:83]: Sure about this? Do have an example file I could look at?

File: configure.in [hunk 2]
```diff
@@ -37,7 +37,7 @@ if test "x$enable_lame" = "xno";then
 fi,DISABLE_LAME=)
 
 PACKAGE=swftools
-VERSION=0.9.2
+VERSION=0.9.2-prezi2
```
Comment (matthiaskramm) [inline configure.in:40]: Could you revert the changes to the version? We'll change this if we ever generate a new .tar.gz.

**Comment by r0mai** [comment]
Sorry, I didn't mean to open this PR just yet to this repository. I will open a cleaned up version later.
